src/core/oip/oip.js

```diff
@@ -289,6 +298,14 @@ class OIP {
     }
     return txids
   }
+
+  /**
+   * Wait for all Confirmation Subscriptions to complete, then resolves the Promise
+   * @return {Promise} Returns a promise that resolves once all subscribed Confirmation callbacks have completed
+   */
+  waitForConfirmations () {
```

Comment:
```suggestion
  async waitForConfirmations () {
```

Reply:
I know that there is a `await waitForConfirmations` elsewhere. Does that require async? I don't remember that.

Reply:
We should be okay to leave off the `async` since it is returning a promise as that makes it inferred.

If you would like we could change it to be `async waitForConfirmations () {`, we would just need to add an `await` before the return as well I believe.